spec/requests/api/my_request/requests/show_spec.rb

```diff
@@ -1,20 +1,20 @@
 # frozen_string_literal: true
 
 RSpec.describe 'GET /api/my_request/requests/:id', type: :request do
-  let!(:requester) { create(:user, email: 'requester@mail.com')}
-  let!(:credentials) { requester.create_new_auth_token }
-  let!(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
-  let!(:myrequest) { create(:request, requester: requester) }
-  let!(:user1) { create(:user) }
-  let!(:user2) { create(:user) }
-  let!(:notmyrequest) { create(:request, requester: user1) }
+  let(:requester) { create(:user, email: 'requester@mail.com')}
+  let(:credentials) { requester.create_new_auth_token }
+  let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
+  let(:my_request) { create(:request, requester: requester) }
+  let(:user_1) { create(:user) }
+  let(:user_2) { create(:user) }
+  let(:not_my_request) { create(:request, requester: user_1) }
```

Comment:
All these renamings, it's  for convention purpose right?

Reply:
Yes, they don't affect anything really